docs/src/pages/components/icons/icons.md

```diff
@@ -7,61 +7,22 @@ components: Icon, SvgIcon
 
 <p class="description">Guidance and suggestions for using icons with Material-UI.</p>
 
-A [system icon](https://material.io/design/iconography/system-icons.html) or UI icon,
-symbolizes a command, file, device, or directory.
-System icons are also used to represent common actions like trash, print, and save,
-and are commonly found in app bars, toolbars, buttons, and lists.
-Google has provided a set of [Material icons](https://material.io/tools/icons/?style=baseline) that follow these guidelines.
+Material-UI provides icons support in three ways.
 
-Material-UI provides two components to render system icons: `SvgIcon` for rendering SVG paths, and `Icon` for rendering font icons.
+- Standardized Material Design icons are exported as React components from `@material-ui/icons` package (Material Icons).
+- `SvgIcon` component as a React wrapper for custom SVG icon is exported from `@material-ui/core`.
+- `Icon` component as a React wrapper for custom Font icons is exported from `@material-ui/core`.
```

Comment:
```suggestion
- The `Icon` component as a React wrapper for custom font icons is exported from `@material-ui/core`.
```

Is it a wrapper?

Reply:
Not sure. Haven't really touched Font icons. Suggest differently.

Reply:
What do you say about
- With the `SvgIcon` component...
- With the `Icon` component...

Reply:
Check latest commit